server/src/configs/Items.yml

```diff
@@ -390,6 +390,8 @@
   craftTasks:
     - Swords
     - IronGear
+  mods:
+    - WeightDecrease
```

Comment:
as mods will be applied at runtime, do we want to handle items that are modded in items config?

Reply:
What was the intention of having them already with mods applied in this config list?

I think these should just be for the pure item configs, and the mod effects would be defined somewhere else.

Reply:
I thought some mobs could drop some upgraded items with mod applied to them, for that we would need it to be in config?

Reply:
It can be done in the drop config on the mob itself and applied in the item constructor.

For example:
```yml
- name: Knight
  ...
  dropList:
    - itemName: HealthPotion
    - itemName: IronBar
    - itemName: IronSword
      rolls: 2
      mods:
        - name: WeightReduction
          level: 3
        - name: DurabilityIncrease
        // No mod level set, use 1 as default.
    - itemName: DungiumBar
      dropRate: 10
```

But in general I think it would be better (for now at least, can expand it when the core of the system is done and we see how it feels in practice) to just keep the mods as separate item drops that are added to the drop list, and not pre-baked onto the dropped items.